src/test/java/com/exasol/adapter/dialects/rls/TableProtectionStatusTest.java

```diff
@@ -0,0 +1,52 @@
+package com.exasol.adapter.dialects.rls;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opentest4j.MultipleFailuresError;
+
+import com.exasol.adapter.dialects.rls.TableProtectionStatus.Builder;
+
+class TableProtectionStatusTest {
+    private Builder builder;
+
+    @BeforeEach
+    void beforeEach() {
+        this.builder = TableProtectionStatus.builder();
+    }
+
+    @Test
+    void testBuildWithRoleProtectedTable() {
+        final String table = "R";
+        this.builder.addRoleProtectedTable(table);
+        assertTableProtectionStatus(table, true, false);
+    }
+
+    private void assertTableProtectionStatus(final String table, final boolean roleProtected,
+            final boolean tenantProtected) throws MultipleFailuresError {
+        final TableProtectionStatus tableProtectionStatus = this.builder.build();
+        assertAll(
+                () -> assertThat("role-protected", tableProtectionStatus.isTableRoleProtected(table),
+                        equalTo(roleProtected)),
+                () -> assertThat("tenant-proteced", tableProtectionStatus.isTableTenantProtected(table),
```

Comment:
proteced -> protected? 
How can the test pass with the typo?

Reply:
Fixed. That text is only debug output.